src/views/GovernanceReviewTeam/Notes/index.tsx

```diff
@@ -106,35 +113,37 @@ const Notes = () => {
         content: values.note
       })
     );
-    notesRefetch();
     await resetForm();
+    if (refetch) {
+      // I was having an issue where sometimes refetch would be undefined.
+      // This may be related to a bug in Apollo Client, so the check here is
+      // to prevent that from breaking the app should it occur.
```

Comment:
This is annoying but [seems to be an issue affecting others as well](https://github.com/apollographql/apollo-client/issues/6816). I didn't spend much time with trying to solve the underlying issue but this check seems to fix things for now.